.github/workflows/generate-desktop-targets.yml

```diff
@@ -22,7 +22,10 @@ env:
   FLEET_DESKTOP_VERSION: 1.37.0
 
 permissions:
-  contents: read
+  contents: write
+  id-token: write
+  attestations: write
+  packages: write
```

Comment:
no individual jobs in this workflow have their own perms, so these should be in effect for all jobs